### PR TITLE
replace all of bitpay/php-client with btcpayserver/php-bitpay-client

### PR DESCRIPTION
**Pull request by psyb0t**
replace all occurrences of bitpay/php-client with btcpayserver/php-bitpay-client for packagist submission and easy use with composer. plus it's less confusing this way.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-    "description": "PHP Library to work with the new cryptographically secure BitPay API",
+    "name": "btcpayserver/php-bitpay-client",
+    "description": "Fork of the PHP Library to work with the new cryptographically secure BitPay API",
     "license": "MIT",
     "minimum-stability": "stable",
     "keywords": ["bitpay", "bitcoin"],
-    "homepage": "https://github.com/bitpay/php-bitpay-client",
+    "homepage": "https://github.com/btcpayserver/php-bitpay-client",
     "support": {
         "email": "support+php@bitpay.com",
```
Comment (NicolasDorier) [inline composer.json:9]: Let's remove the email

File: composer.json [hunk 2]
```diff
@@ -1,14 +1,14 @@
 {
-    "name": "bitpay/php-client",
-    "description": "PHP Library to work with the new cryptographically secure BitPay API",
```
Comment (NicolasDorier) [inline composer.json:3]: `PHP library to work with BTCPay Server` the API is now diverging from Bitpay, so I think we can stop mentioning them.

File: build/stub.php [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <?php
 /**
- * @license Copyright 2011-2015 BitPay Inc., MIT License 
- * see https://github.com/bitpay/php-bitpay-client/blob/master/LICENSE
+ * @license Copyright 2011-2015 BitPay Inc., MIT License
```
Comment (NicolasDorier) [inline build/stub.php:3]: Still Bitpay in the license

File: build/rulesets/phpmd.xml [hunk 4]
```diff
@@ -2,7 +2,7 @@
 <!--
 /**
  * @license Copyright 2011-2015 BitPay Inc., MIT License
```
Comment (NicolasDorier) [inline build/rulesets/phpmd.xml:4]: Still Bitpay in the license

File: build/phpunit.xml.dist [hunk 5]
```diff
@@ -2,7 +2,7 @@
 <!--
 /**
  * @license Copyright 2011-2015 BitPay Inc., MIT License
- * see https://github.com/bitpay/php-bitpay-client/blob/master/LICENSE
+ * see https://github.com/btcpayserver/php-bitpay-client/blob/master/LICENSE
```
Comment (NicolasDorier) [inline build/phpunit.xml.dist:5]: Still Bitpay in the license

**Comment by psyb0t** [comment]
i accidentally messed up my fork...
will close this and do it again from the start. sry

**Pull request closed without merging**